### PR TITLE
Make a nicer interface for token loading

### PR DESCRIPTION
**Pull request by imbstack**
This is what we talked about on irc yesterday.

### CODE REVIEW

**Comment by jonasfj** [comment]
readme needs update too.. otherwise good

**Comment by gregarndt** [comment]
Ok, if jonas is happy I'm happy :)

File: src/client.js [hunk 1]
```diff
@@ -11,21 +11,28 @@ class StatsumClient extends events.EventEmitter {
   /**
    * Create a statsum client.
    *
-   * Óptions:
+   * Configurer: A function that returns an object like
```
Comment (gregarndt) [inline src/client.js]: I'm not sure I understand this statement... "returns an object like"

Comment (imbstack) [inline src/client.js]: Oh yeah, that's definitely not worded well. I'll change to

```
A function that must return an object with these specific keys
```

**Pull request merged**